product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx

```diff
@@ -1,37 +1,40 @@
 ---
-title: "Responsibility model"
+title: "Shared responsibilities"
```

Comment:
```suggestion
title: "Shared security responsibilities"
```

Reply:
We shouldn't limit this to security. Performance is an important part of the shared responsibility model, for example.